Dockerfile

```diff
@@ -1,5 +1,12 @@
-FROM python:3.9.13 AS base
+FROM node:13.13.0 as node
+ENV NODE_ENV=development
```

Comment:
Is the intent to override this value in the pipeline definition? Otherwise the produced assets won't be minified.

Reply:
Oh good catch. This was a failed threadchase from debugging yesterday. This should be 'production'.